recipe/meta.yaml

```diff
@@ -18,27 +18,39 @@ requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
+    - poetry-core
   run:
     - python
+    - leb128
+    - pytz
+    - lz4
+    - python-zstd
+    - tzlocal
+    - ciso8601
+  run_constrained:
+    - clickhouse-cityhash
 
 test:
   imports:
-    - {{ name.replace('-', '.') }}
+    - asynch
```

Comment:
There are several other imports that could be checked under `asynch.proto`.

Reply:
Are you suggesting to also add `async.proto` to test imports?

Reply:
Yes, as well as `asynch.proto.columns`, `asynch.proto.compression`, `asynch.proto.settings`, `asynch.proto.streams`, and `asynch.proto.utils`. This ensures that there are not imports in submodules that are not accounted for.

Reply:
Where are you seeing these?

Reply:
They generally follow the folder structure within the asynch python module. Sometimes, folders within a module are not submodules and you can have submodules without using a folder/__init__.py structure but this is the PEP-compliant way of do so these days.
https://github.com/long2ice/asynch/tree/v0.2.4/asynch/proto

Reply:
@bwingconda I'm not quite following - what do these import tests catch exactly? I.e. what kind of mistake might we have made through packaging this, in which the imports are dropped?

Reply:
It's a validation check on the structure of the module to help packagers who may not always look at the source code changes when updating a version. In this way, it's possible to detect major changes within the package at package time rather than finding out a submodule was renamed and an import is broken only when it's used down the line.

Definitely not critical, but fairly easy to do and has the potential to catch issues earlier rather than later.